java-engine/src/main/java/io/getunleash/engine/CustomStrategiesEvaluator.java

```diff
@@ -17,7 +17,7 @@
 
 class CustomStrategiesEvaluator {
   private static final Logger log = LoggerFactory.getLogger(CustomStrategiesEvaluator.class);
-  static final String EMPTY_STRATEGY_RESULTS = "{}";
+  static final Map<String, Boolean> EMPTY_STRATEGY_RESULTS = Map.of();
```

Comment:
The public API here now returns a Map<String, Boolean> instead of a JSON string, which isn't helpful for flat buffers